drivers/i2c/Kconfig.tca9546a

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 Innoseis BV
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig I2C_TCA9546A
+	bool "I2C addressable switch"
+	help
+		Enable TCA9546A i2C bus switch
```

```suggestion
	  Enable TCA9546A i2C bus switch
```

This one was not quite updated as suggested. Please keep it consistent with the rest of the kconfig help throughout the tree by using a tab and two spaces instead of two tabs. I know it's weird, but see e.g. Kconfig.zephyr; we inherit this from Linux and are generally consistent about it.